### PR TITLE
Cut v0.16.0

### PR DESCRIPTION
**Pull request by mxinden**
In case there are no further bugs reported I am planning to cut this release at the beginning of next week. Let me know what you think.

### CODE REVIEW

**Comment by brancz** [comment]
sounds good to me!

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,12 +20,23 @@ deprecated. API v1 will be removed with Alertmanager release v0.18.0.
...
 * [ENHANCEMENT] Make OpsGenie API Key a templated string (#1594)
 * [ENHANCEMENT] Add name, value and SlackConfirmationField to action in Slack notifier (#1557)
 * [ENHANCEMENT] Show more alert information on silence form and silence view pages (#1601)
 * [ENHANCEMENT] Add cluster peers DNS refresh job (#1428)
+* [BUGFIX] Fix unmarshalling of secret URLs in config (#1663)
+* [BUGFIX] Do not write groupbyall and groupby when marshaling config (#1665)
+* [BUGFIX] Make a copy of firing alerts with EndsAt=0 when flushing (#1686)
+* [BUGFIX] Respect regex matchers when recreating silences in UI (#1697)
+* [BUGFIX] Change DefaultGlobalConfig to a function in Alertmanager configuration (#1656)
+* [BUGFIX] [amtool] Support for assuming first label is alertname in silence add and query (#1693)
```
[the remark this answers is not in the document]
Comment (mxinden) [reply CHANGELOG.md]: Good idea. Thanks.

**Comment by mxinden** [comment]
github.com/coreos/prometheus-operator test suit succeeded as well: https://travis-ci.org/coreos/prometheus-operator/builds/482346735.

**Comment by stuartnelson3** [comment]
haven't had any issues at sc

**Comment by mxinden** [comment]
> haven't had any issues at sc

Cool, thanks for testing!

**Pull request merged**